### DISCUSSION TITLE
Set stat from script

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#1993] [Q&A]
How do i set a stat from a script?
In kiddion it's like this
```
some_script = script("script_name")
some_script:set_int(123, 456)
```

But i can't figure out how to do it in Yim.
Is it even currently possible?

**Comment by ScottiePimp** [comment]
script_local

**Reply by pierrelasse** [reply]
what do you mean? could you please give me an example setting a stat of fmmissipncontroller script?

**Reply by binded-zz** [reply]
some scripts some use each other idk whats really in each of them mostly CN lang so you have to convert them 
or find someone that has some of the links have those 

https://github.com/YimMenu-Lua
https://github.com/sch-lda/SCH-LUA-YIMMENU
https://github.com/L7NEG/Ultimate-Menu

Heist Lua by Wang (ZH_CN) https://github.com/wangzixuank/YimMenu-HeistLua
GTA5线上小助手Lua仓库 (ZH_CN) GTA5OnlineTools' Online Lua Repo https://github.com/CrazyZhang666/GTA5OnlineLua
RScript (ZH_CN) https://github.com/TCRoid/YimMenu-Lua-RScript
Alestarov_Menu (EN) https://github.com/Alestarov/YimMenu-lua-script-Alestarov_Menu

**Comment by binded-zz** [comment]
also forgot to say 
https://github.com/Harmless05/harmless-lua
he has about 5 or 6 small scripts that you might be able to look at as well

**Comment by pierrelasse** [comment]
K. I made this function which works. Thx for ur help
```
function isScriptRunning(name)
        return SCRIPT.GET_NUMBER_OF_THREADS_RUNNING_THE_SCRIPT_WITH_THIS_HASH(joaat(name)) ~= 0
    end
```
